packages/ui-library/src/components/checkbox/index.test.ts

```diff
@@ -34,6 +34,20 @@ describe('blr-checkbox', () => {
     expect(className).to.contain('input-control');
   });
 
+  it(' error Icon is set to undefined', async () => {
```

Comment:
Please remove the extra space before the _error Icon_. Also would be nice to have it more contextual while reading by rephrasing it to something like **it('sets the error Icon to undefined')**.